### DISCUSSION TITLE
Restart and running slowly

### DISCUSSION THREAD

**Discussion by Sahinoglu0** [mashb1t/Fooocus#78] [Q&A] [answered]
https://github.com/mashb1t/Fooocus/discussions/77

This solved the problem for the error I was getting.
I deleted everything and installed it using terminal and commands. Now everything works. But I don't know how to turn it back on after I closed it. What is the restart command?

Another problem I am having is that content is rendering very slowly for the system I have. Is there a way to fix this?

**Comment by mashb1t** [comment, marked as answer]
Your run.bat shound contain these 2 lines of code:

```
.\python_embeded\python.exe -s Fooocus\entry_with_update.py --directml
pause
```

your install_and_run.bat all of the mentioned lines in the readme.

> content is rendering very slowly for the system I have

your hardware isn't the most powerful, and as "very slow" is vague i can't judge if this is the case. Probably there's not many things you can do to improve the performance (if any).

**Reply by Sahinoglu0** [reply]
I already know that my hardware is not the most powerful. But you told me that 1gb vram is used. But I have 8gb vram. I changed the specific parameter about vram in the model_management file. Now 8gb vram is used. It is a little better.
